doc/abyss-pe.1

```diff
@@ -49,7 +49,7 @@ files containing single-end reads
 maximum number of branches of a bubble [2]
 .TP
 .B b
-maximum length of a bubble (bp) [10000]
+maximum length of a bubble (bp) [3*k]
```

Comment:
Thanks @sjackman for the detailed explanation! I think it is very bad that running the program without specifying a value for an option may give different results from specifying the default value for it.

Maybe an interim solution may be to pass the value of `b` only to `PopBubbles` (default `10000`) and leave `ABYSS` to always run with its default of `3*k`, what do you think?